client/src/app/components/login-form/login-form.component.html

```diff
@@ -0,0 +1,38 @@
+<form (ngSubmit)="onSubmit()" class="login-form">
+  <ion-item>
+    <ion-label position="stacked">E-mail</ion-label>
+    <ion-input
+      name="email"
+      type="email"
+      required
+      inputmode="email"
+      pattern="[a-zA-Z0-9\.]+@[a-zA-Z0-9]+\.[a-zA-Z0-9]+"
```

Comment:
If we use `type=email`, I don't think this pattern is required.
```suggestion
```

Reply:
with just `type email` the field is green with any string

Reply:
Technically.... almost any string IS a valid email-address. :D 

There are a lot of more specifics on: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email#validation
But not all of these things apply in the same way to these Ionic/Angular-components as well, so nevermind for now. ;)